src/uct/ib/base/ib_md.c

```diff
@@ -1517,8 +1514,11 @@ uct_component_t uct_ib_component = {
 
 void UCS_F_CTOR uct_ib_init()
 {
+    UCS_MODULE_FRAMEWORK_DECLARE(uct_ib);
     ssize_t i;
 
+    UCS_MODULE_FRAMEWORK_LOAD(uct_ib, 0);
```

Comment:
I think the load might need to be placed after line 1523 below, to preserve current ordering for component and ib ops.

Reply:
fixed